packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -160,7 +160,24 @@ export default class Proxy {
       'vm-data/xoa-updater-channel': JSON.stringify(this._xoProxyConf.channel),
     })
 
-    return xapi.rebootVm(vmUuid)
+    try {
+      await xapi.rebootVm(vmUuid)
+    } catch (error) {
+      if (error.code !== 'VM_BAD_POWER_STATE') {
+        throw error
+      }
+
+      await xapi.startVm(vmUuid)
+    }
+
+    await xapi._waitObjectState(
+      vmUuid,
+      vm => extractIpFromVmNetworks(vm.$guest_metrics?.networks) !== undefined
+    )
```

Comment:
Why is this unnecessary now?

Reply:
To be able to fetch updates. See  https://github.com/vatesfr/xen-orchestra/pull/5167/files#diff-5c00d01ffc1fb5facb3a83a9a7688111R202